portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -48,4 +46,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         response.setContentType("application/json;");
         response.getWriter().println(json);
     }
+
+    @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String comment = request.getParameter("comment");
+      messages.add(comment);
+    
+      // Redirect back to the HTML page.
+      response.sendRedirect("/index.html#add-comments");
+  }
```

Comment:
This should be indented 1 level further.

Reply:
done!